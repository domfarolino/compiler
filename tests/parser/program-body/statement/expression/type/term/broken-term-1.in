program dominic is

  integer dominic;
  string farolino;

begin

  dominic := 10 * farolino;

end program.
