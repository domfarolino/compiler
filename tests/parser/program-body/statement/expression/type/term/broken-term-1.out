[1;31mLine 8: [Term]: integer and string cannot be multiplied or divided
Line 8: Error parsing program body
[0m